src/Views/FirmwareView.vala

```diff
@@ -216,7 +215,7 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
     private void header_rows (Widgets.FirmwareUpdateRow row1, Widgets.FirmwareUpdateRow? row2) {
         if (row2 == null && row1.is_updatable) {
             var header = new FirmwareHeaderRow (
-                dngettext (GETTEXT_PACKAGE, "%u Update Available", "%u Updates Available", num_updates).printf (num_updates)
+                dngettext (GETTEXT_PACKAGE, "%u New", "%u New", num_updates).printf (num_updates)
```

Comment:
This is not more clear. "1 New" what? These are available updates

Reply:
I imagined this was under some other title since the original is just "update".

Reply:
```suggestion
                dngettext (GETTEXT_PACKAGE, "%u available firmware upgrade", "%u available firmware upgrades", num_updates).printf (num_updates)
```

Reply:
I am a fan of "One available", but don't know if that causes issues for any translators.

Reply:
This is used in the UI as a section header, so it is intended to be title case. Since this whole pane deals with firmware, we don't need to specify that these are firmware updates. I don't think changing this string makes anything more clear